build/media-suite_compile.sh

```diff
@@ -1816,11 +1817,12 @@ if { { [[ $ffmpeg != no ]] && enabled vulkan; } || ! mpv_disabled vulkan; } &&
         do_install d3d{kmthk,ukmdt}.h include/
     cd_safe "$(get_first_subdir -f)"
     do_print_progress "Building Vulkan-Loader"
-    CFLAGS+=" -DSTRSAFE_NO_DEPRECATE" do_cmakeinstall -DBUILD_TESTS=OFF -DCMAKE_SYSTEM_NAME=Windows -DUSE_CCACHE=OFF \
-    -DCMAKE_ASM_COMPILER="$(command -v nasm.exe)" -DVULKAN_HEADERS_INSTALL_DIR="$LOCALDESTDIR" \
+    CC=gcc CXX=g++ CFLAGS+=" -DSTRSAFE_NO_DEPRECATE" do_cmakeinstall -DBUILD_TESTS=OFF \
+    -DCMAKE_SYSTEM_NAME=Windows -DUSE_CCACHE=OFF \
```

Comment:
btw, why do we set `CMAKE_SYSTEM_NAME` here? Isn't it usually `Windows` either way?

https://cmake.org/cmake/help/latest/variable/CMAKE_SYSTEM_NAME.html

https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_SYSTEM_NAME.html

Reply:
No idea, probably cargo-culted from somewhere.